cypress/test/TOC.cy.js

```diff
@@ -0,0 +1,70 @@
+import { mount } from '@cypress/react';
+import TOC from '../../components/TOC';
+
+describe('TOC', () => {
+  const toc = [
+    {
+      lvl: 1,
+      content: '<h1>Introduction</h1>',
+      slug: 'introduction',
+      slugWithATag: 'introduction',
+    },
+    {
+      lvl: 2,
+      content: '<h2>Getting Started</h2>',
+      slug: 'getting-started',
+      slugWithATag: 'getting-started',
+    },
+    {
+      lvl: 2,
+      content: '<h2>Installation</h2>',
+      slug: 'installation',
+      slugWithATag: 'installation',
+    },
+    {
+      lvl: 3,
+      content: '<h3>Step 1: Download</h3>',
+      slug: 'step-1-download',
+      slugWithATag: 'step-1-download',
+    },
+    {
+      lvl: 3,
+      content: '<h3>Step 2: Install Dependencies</h3>',
+      slug: 'step-2-install-dependencies',
+      slugWithATag: 'step-2-install-dependencies',
+    },
+  ];
+  it('renders the TOC correctly with empty content', () => {
+    mount(<TOC toc={[]} />);
+
+    cy.get('.hidden').should('not.exist');
```

Comment:
use data-tested @reachaadrika

Reply:
@reachaadrika Kindly address this comment.

Reply:
@akshatnema test-id is not working , here tried to use both the divs , since I am only checking for hidden which is an optional css